a10_neutron_lbaas/v2/handler_listener.py

```diff
@@ -101,14 +133,21 @@ def _set(self, set_method, c, context, listener):
                 status=status,
                 autosnat=c.device_cfg.get('autosnat'),
                 ipinip=c.device_cfg.get('ipinip'),
-                axapi_body=vport_meta)
+                template_client_ssl=template_args.get("template-client-ssl"),
+                template_server_ssl=template_args.get("template-server-ssl"),
+                axapi_body=vport_meta,
+                **template_args)
         except acos_errors.Exists:
             pass
 
     def _set_terminated_https_values(self, listener, c, cert_data):
         is_success = False
+        container = None
+        bindings = []
```

Comment:
One of these days, I'll get the "declare variables before you use them to ensure the code reads verbosely" pattern out of my head.  Yep, it's effectively no-op and only lives there for my sanity.  I'll kill it.
________________________________
From: Cedric Shock [notifications@github.com]
Sent: Tuesday, June 21, 2016 9:18 AM
To: a10networks/a10-neutron-lbaas
Cc: Michael Durrant; Author
Subject: Re: [a10networks/a10-neutron-lbaas] Support for SSL Cert binding in listeners (#354)


In a10_neutron_lbaas/v2/handler_listener.py<https://github.com/a10networks/a10-neutron-lbaas/pull/354#discussion_r67901521>:

>          except acos_errors.Exists:
>              pass
>
>      def _set_terminated_https_values(self, listener, c, cert_data):
>          is_success = False
> +        container = None
> +        bindings = []


Trivial: these changes do nothing

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/a10networks/a10-neutron-lbaas/pull/354/files/5dc9ce576a8e1982a906a41b5c1d0b028658194c#r67901521>, or mute the thread<https://github.com/notifications/unsubscribe/AJlobCfh9fu4WTbyjSrHMXOo-RpyWz7Vks5qOA7hgaJpZM4I6aIY>.